### PR TITLE
TC06 - User is redirected to Home page after logging out

### PR DESCRIPTION
**Pull request by cvan8397**

### CODE REVIEW

**Comment by thanhlelgg** [comment]
please resolve conflict

File: src/main/java/page_objects/GeneralPage.java [hunk 1]
```diff
@@ -11,6 +11,7 @@
     private final Tab logOutTab = new Tab(By.cssSelector("a[href='/Account/Logout']"));
     private final Tab changePwdTab = new Tab(By.cssSelector("a[href='/Account/ChangePassword.cshtml']"));
     private final Tab MyTicketTab = new Tab(By.cssSelector("a[href='/Page/ManageTicket.cshtml']"));
```
Comment (tanmle) [inline src/main/java/page_objects/GeneralPage.java]: naming convention

File: src/main/java/page_objects/GeneralPage.java [hunk 2]
```diff
@@ -11,6 +11,7 @@
...
+    private final Tab ContactTab = new Tab(By.cssSelector("a[href='/Page/Contact.cshtml']"));
```
Comment (tanmle) [inline src/main/java/page_objects/GeneralPage.java]: naming convention

File: src/main/java/page_objects/GeneralPage.java (hunk 1 above)
Comment (tanmle) [reply src/main/java/page_objects/GeneralPage.java]: this is unacceptable

File: src/main/java/page_objects/GeneralPage.java (hunk 2 above)
Comment (tanmle) [reply src/main/java/page_objects/GeneralPage.java]: this is unacceptable

File: src/main/java/page_objects/GeneralPage.java (hunk 1 above)
Comment (tanmle) [reply src/main/java/page_objects/GeneralPage.java]: I told you about this at the very beginning of class and now you make this mistake

File: src/main/java/page_objects/GeneralPage.java (hunk 2 above)
Comment (tanmle) [reply src/main/java/page_objects/GeneralPage.java]: I told you about this at the very beginning of class and now you make this mistake

**Review by tanmle — CHANGES_REQUESTED** [review]

**Comment by cvan8397** [comment]
@tanmle I have fixed my code. Please review it again. Sorry about my mistake

File: src/main/java/page_objects/HomePage.java [hunk 3]
```diff
@@ -1,4 +1,12 @@
 package page_objects;
 
+import elements.Label;
+import org.openqa.selenium.By;
+
 public class HomePage extends GeneralPage {
+    private final Label lblWelcomeRailWay = new Label(By.cssSelector("#content>h1"));
+
+    public String getWelcomeRailWayText() {
```
Comment (tanmle) [inline src/main/java/page_objects/HomePage.java]: this is page header or page title, not RailwayText

File: src/test/java/tests/TC06_UserIsRedirectedToHomePageAfterLoggingOut.java [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+
+    @Test(description = "User is redirected to Home page after logging out")
+    public void testCase06() {
+        Account account = new Account(Constant.USERNAME, Constant.PASSWORD);
+        homePage.gotoLoginPage();
+        loginPage.login(account);
+        loginPage.gotoContactPage();
+        contactPage.logout();
+
+        String actualResult = homePage.getWelcomeRailWayText();
```
Comment (tanmle) [inline src/test/java/tests/TC06_UserIsRedirectedToHomePageAfterLoggingOut.java]: this should be actualTitle or actualyHeader

File: src/test/java/tests/TC06_UserIsRedirectedToHomePageAfterLoggingOut.java [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+        String expectedResult = "Welcome to Safe Railway";
```
Comment (tanmle) [inline src/test/java/tests/TC06_UserIsRedirectedToHomePageAfterLoggingOut.java]: same here, expectedTitle or expectedHeader

**Review by tanmle — CHANGES_REQUESTED** [review]

**Comment by cvan8397** [comment]
@tanmle I have fixed the variable's name. Please review it again

File: src/test/java/tests/TC06_UserIsRedirectedToHomePageAfterLoggingOut.java (hunk 5 above)
Comment (cvan8397) [reply src/test/java/tests/TC06_UserIsRedirectedToHomePageAfterLoggingOut.java]: I have fixed

File: src/main/java/page_objects/HomePage.java (hunk 3 above)
Comment (cvan8397) [reply src/main/java/page_objects/HomePage.java]: I have fixed

File: src/test/java/tests/TC06_UserIsRedirectedToHomePageAfterLoggingOut.java (hunk 4 above)
Comment (cvan8397) [reply src/test/java/tests/TC06_UserIsRedirectedToHomePageAfterLoggingOut.java]: I have fixed

File: src/main/java/page_objects/HomePage.java [hunk 6]
```diff
@@ -1,4 +1,12 @@
 package page_objects;
 
+import elements.Label;
+import org.openqa.selenium.By;
+
 public class HomePage extends GeneralPage {
+    private final Label lblWelcomeRailWay = new Label(By.cssSelector("#content>h1"));
+
+    public String getWelcomeRailWayHeader() {
```
Comment (tanmle) [inline src/main/java/page_objects/HomePage.java]: this should be just getHeader()

Comment (tanmle) [reply src/main/java/page_objects/HomePage.java]: remember to update the element name as well.

Comment (cvan8397) [reply src/main/java/page_objects/HomePage.java]: I have changed it

**Review by tanmle — APPROVED** [review]

**Pull request merged**